doc/apidoc/smac.facade.smac_facade.rst

```diff
@@ -0,0 +1,7 @@
+smac\.facade\.smac\_facade module
+=================================
+
```

Comment:
@shukon could you please update the return value of `get_trajectory` in the docstring? This function actually returns a named tuple now.